components/tyk-enterprise-portal/templates/service-enterprise-portal.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: enterprise-portal-svc-{{ include "tyk-enterprise-portal.fullname" . }}
+  labels:
+    app: enterprise-portal-svc-{{ include "tyk-enterprise-portal.fullname" . }}
+    chart: {{ include "tyk-enterprise-portal.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+{{- if .Values.service.annotations }}
+  annotations:
+  {{- range $key, $value := .Values.service.annotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+{{- end }}
+spec:
+  ports:
+  - port: {{ .Values.service.port }}
+    targetPort: {{ .Values.containerPort }}
+    protocol: TCP
+    name: http
+  type: {{ .Values.service.type }}
+  selector:
+    app: enterprise-portal-{{ include "tyk-enterprise-portal.fullname" . }}
```

This label is incorrect. It should be
```suggestion
    app: {{ include "tyk-enterprise-portal.fullname" . }}
```

---

Fixed